Jenkinsfile

```diff
@@ -0,0 +1,124 @@
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the GOUpdate step of Reactome's release.
```

Comment:
Is GOUpdate the name of the Jenkins step?  If so, it might be good to put quotes around it so it's clear it's a name and there shouldn't be a space between "GO" and "Update"